content/master/concepts/pods.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Crossplane Pods
+weight: 1
+description: Background on the components installed with Crossplane and their functions.
+---
+
+The base Crossplane installation consists of two Pods, the `crossplane` pod and
+the `rbac-manager` pod. Both pods install in the `crossplane-system`
+namespace by default. 
+
+
+## Crossplane pod
+
+### Init pod
+Before starting the core Crossplane pod an _init_ Pod runs. The init pod is
+responsible for setting parameters related to the core Crossplane pod and
+installing the core Crossplane 
+[Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+(`CRDs`).
+
+The settings the init pod sets include installing Provider or Configuration 
+packages with Crossplane, customizing the namespace Crossplane installs in and 
+defining webhook configurations. 
+
+The core CRDs installed by the init pod include: 
+* Composite Resource Definitions, Compositions, Configurations and Providers
+* `locks` to manage package dependencies
+* `controllerconfigs` to apply settings to installed Providers
+* `storeconfigs` for connecting external secret stores like 
+[Hashicorp Vault](https://www.vaultproject.io/)
+
+{{< hint "note" >}}
+
+The [Install Crossplane]({{< ref "../software/install" >}}) section has more
+information about customizing the Crossplane install.
+{{< /hint >}}
+
+The init pod completes and starts the Crossplane core pod. 
```

Comment:
What happens to the init pod after completion? Does the end user have to care about it at all after completion? Does it also run in the `crossplane-system` namespace?

Reply:
I added more information about viewing the init container in `k get pods` and that starts the core container automatically.